src/windows/scanner.cc

```diff
@@ -146,29 +156,35 @@ static drivelist::Code ScanDisks(drivelist::com::Connection *const connection,
 static drivelist::Code
 ScanMountpoints(std::vector<drivelist::mountpoint_s> *const output) {
   wchar_t systemDriveLetter;
+  drivelist::Debug("Getting system volume drive letter");
   HRESULT result = drivelist::volume::GetSystemVolume(&systemDriveLetter);
   if (FAILED(result))
     return InterpretHRESULT(result);
 
```

Comment:
Perhaps it'd have been useful here to actually log _which_ `systemDriveLetter` we got? ;)